catalog/queries/lookup/aat.rq

```diff
@@ -0,0 +1,49 @@
+PREFIX aat: <http://vocab.getty.edu/aat/>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX gvp: <http://vocab.getty.edu/ontology#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX skosxl: <http://www.w3.org/2008/05/skos-xl#>
+
+CONSTRUCT {
+    ?uri a skos:Concept ;
+        skos:prefLabel ?prefLabel ;
+        skos:altLabel ?altLabel ;
+        skos:scopeNote ?scopeNote ;
+        skos:broader ?broader_uri ;
+        skos:narrower ?narrower_uri .
+    ?broader_uri skos:prefLabel ?broader_prefLabel .
+    ?narrower_uri skos:prefLabel ?narrower_prefLabel .
+}
+WHERE {
+    # For example:
+    # Bicycles: <http://vocab.getty.edu/aat/300212636>
+    # Wars: <http://vocab.getty.edu/aat/300055314>
+    VALUES ?uri { ?uris }
```

Comment:
To discuss: `?uris` is the keyword that we can use to inject the (list of) URIs into the query. Can Comunica handle this?

Reply:
Comunica doesn’t handle this for us; just like with the search queries, we’ll do the search/replace ourselves.

Reply:
👍🏻